pages/linux/flameshot.md

```diff
@@ -4,30 +4,34 @@
 > Supports basic image editing, such as text, shapes, colors, and imgur.
 > More information: <https://flameshot.org>.
 
-- Launch flameshot with a simpler interactive mode:
+- Create a fullscreen screenshot:
 
-`flameshot launcher`
+`flameshot full`
 
-- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot:
+- Create a screenshot interactively:
 
 `flameshot gui`
 
-- Take a full screenshot (all monitors):
+- Create a screenshot and save it to a specific path:
 
-`flameshot full`
+`flameshot gui --path {{path/to/directory}}`
+
+- Create a screenshot interactively in a simplified mode:
 
-- Take a screenshot from monitor 1:
+`flameshot launcher`
+
+- Create a screenshot from a specific monitor:
 
-`flameshot screen --number {{1}}`
+`flameshot screen --number {{5}}`
```

Comment:
How many screens do you have? :laughing: 
```suggestion
`flameshot screen --number {{2}}`
```

Reply:
I have one screen. 😆